src/CommunityToolkit/Xamarin.CommunityToolkit/Views/BaseTemplatedView.shared.cs

```diff
@@ -9,9 +9,7 @@ namespace Xamarin.CommunityToolkit.UI.Views.Internals
 	/// <typeparam name="TControl">The type of the control that this template will be used for</typeparam>
 	public abstract class BaseTemplatedView<TControl> : TemplatedView where TControl : View, new()
 	{
-		TControl? control;
-
-		protected TControl Control => control ?? throw new NullReferenceException();
+		protected TControl? Control { get; private set; }
```

Comment:
I'm not sure if this is a breaking change, can you confirm that before merge?

Reply:
It's not a breaking change. Basically, nothing changed